src/import/dispatch.ts

```diff
@@ -30,6 +31,12 @@ async function matchImporter(importSpec: ImportSpec, argv: any): Promise<ImportB
     return new ImportKubernetesApi(k8s);
   }
 
+  const helm = importSpec.source.split(':')[0];
+
+  if (helm === 'helm') {
+    return new ImportHelm(importSpec);
```

Comment:
For consistency with other importers:

 ```suggestion
    return ImportHelmChart.fromSpec(importSpec);
```